### DISCUSSION TITLE
Chapter 10 - Sensory Delay

### DISCUSSION THREAD

**Discussion by argunaykut** [softmatterlab/SOCS#19] [General]
Here, you can ask questions and discuss issues about the simulations of Chapter 10 - Sensory Delay. 
We encourage you to participate in these discussions and engage with our community.

**Comment by acallegar** [comment]
Hi, for the students asking about exercise 10.1 and 10.2

--- a possible choice for the simulation **parameters**
I0 = 10;          %
V0 = 10;         %
Vinf = 0.01;    %
tau = 2;          % 
L = V0 * tau;    % persistence length
dt = 0.01 * tau;  % time step
N_iterations = 100*round(tau/dt);   % 100 times tau

--- Hint for **10.1** : how to show that the robot spends more time in the spots with high intensities: 
_try using histograms_

--- Hint for **10.2** : how to show whether the robot trajectory is affected by the traveling wave: 
_try using histograms, and compare to a trajectory that is static and with the same rotational noise_

**Reply by acallegar** [reply]
![medium_ctau_hist](https://user-images.githubusercontent.com/66014271/204053880-e0f5ea81-1ed5-4fdb-8766-4464809e7768.png)

Example of histogram for comparison wave vs static pattern

**Reply by acallegar** [reply]
Example of trajectory comparison (the pattern below the trajectories is the static one, i.e., the reference for the cyan trajectory)
the trajectory in white is the one for the moving pattern (wave traveling from left to right)

![medium_ctau_traj](https://user-images.githubusercontent.com/66014271/204054040-d1c29fbf-7a53-4139-93c1-a0d94a0ba282.png)

**Comment by C4illin** [comment]
The example code "multiple_robots,.ipynb" seems to be wrong and unfinished?

**Reply by acallegar** [reply]
Hi, we will check the code.
